src/any_llm/any_llm.py

```
@@ -1351,14 +1364,148 @@ async def aresponses(
             msg = "Provider doesn't support responses."
             raise NotImplementedError(msg)
         msg = "Subclasses must implement _aresponses method"
         raise NotImplementedError(msg)
 
+    @overload
+    def retrieve_response(
+        self,
+        response_id: str,
+        *,
+        stream: Literal[True],
+        **kwargs: Any,
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)